src/components/Widget/index.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import {Heading, Paragraph} from 'egghead-ui'
+
+export default ({children, title, description}) => (
+  <section className='mb4'>
+    <Heading level='4'>
+      {title}
+    </Heading>
+    {description
+      ? <Paragraph type='small'>
+          {description}
+        </Paragraph>
+      : null
+    }
+    <div className='bg-white shadow-1 br2'>
+      {children}
+    </div>
+  </section>
+)
```

Comment:
Used a lot now. Gives a title, optional description, and renders children; children are often wrapped in `List`:
![image](https://cloud.githubusercontent.com/assets/5497885/23816759/bd077af4-05ab-11e7-9ecc-f44b76f9669e.png)